start_ocean.sh

```diff
@@ -1,15 +1,19 @@
 #!/usr/bin/env bash
 
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+COMPOSE_DIR="${DIR}/compose-files"
 # Must be set to true for the first run, change it to "false" to avoid migrating the smart contracts on each run.
 export DEPLOY_CONTRACTS="true"
 # Ganache specific option, these two options have no effect when not running ganache-cli
-export GANACHE_DATABASE_PATH="."
+export GANACHE_DATABASE_PATH="${DIR}"
 export REUSE_DATABASE="false"
 # Specify which ethereum client to run or connect to: kovan, ganache, or ocean_poa_net_local
 export KEEPER_NETWORK_NAME="ganache"
-export ARTIFACTS_FOLDER=~/.ocean/keeper-contracts/artifacts
+export ARTIFACTS_FOLDER=$HOME/.ocean/keeper-contracts/artifacts
+export BRIZO_ENV_FILE=$DIR/brizo.env
```

Comment:
BRIZO_ENV_FILE is not added to brizo's docker-compose file.

Reply:
Thanks! Solved